ts-rs/src/export.rs

```diff
@@ -135,7 +136,15 @@ pub(crate) fn export_to<T: TS + ?Sized + 'static, P: AsRef<Path>>(
         std::fs::create_dir_all(parent)?;
     }
     let lock = FILE_LOCK.lock().unwrap();
-    std::fs::write(path.as_ref(), buffer)?;
+    {
+        // Manually write to file & call `sync_data`. Otherwise, calling `fs::read(path)` 
+        // immediately after `T::export()` might result in an empty file.
+        use std::io::Write;
+        let mut file = File::create(path)?;
+        file.write_all(buffer.as_bytes())?;
+        file.sync_data()?;
+    }
```

Comment:
So that's why I was getting some weird empty files when adding `#[ts(export)]` to some tests xD

Reply:
possibly :D  
I noticed it in `tests/imports.rs`, where line 68 would panic with an empty string.